recipes/documenteer/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "documenteer" %}
+{% set version = "0.5.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "06fbceb7324ed3b00f012b90b2da0e0aff9faab7957d6d7c5f09cc5a3069c9cf"
+
+build:
+  skip: true  # [py27]
+  number: 0
+  entry_points:
+    - stack-docs = documenteer.stackdocs.stackcli:main
+    - package-docs = documenteer.stackdocs.packagecli:main
+    - build-stack-docs = documenteer.stackdocs.build:run_build_cli
+    - refresh-lsst-bib = documenteer.bin.refreshlsstbib:run
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
+    - click
+    - gitpython
+    - breathe
+    - lsst-dd-rtd-theme ==0.2.2
+    - lsst-sphinx-bootstrap-theme >=0.2.0,<0.3.0
+    - numpydoc >=0.8.0,<0.9.0
+    - pyyaml
+    - requests
+    - sphinx >=1.7.0,<1.8.0
+    - sphinx-prompt
+    - sphinx-automodapi >=0.7,<0.8
+    - sphinx-click
+    - sphinx-jinja ==1.1.0
+    - sphinxcontrib-autoprogram >=0.1.5,<0.2.0
+    - sphinxcontrib-bibtex ==0.4.0
```

Comment:
Please remove the exact pins. This makes the package hard to install for other users. To be honest, probably none of these pins should be here